tests/unit/ManualBusinessInfo.spec.ts

```diff
@@ -35,29 +38,40 @@ describe('Manual BusinessInfo component', () => {
       homeIncorporationDate: '',
       homeLegalName: '',
       mode: 'MANUAL'
-    }
-    const getters = {
-      isContinuationInAffidavitRequired: () => true,
-      getExistingBusinessInfo: () => businessMock
-    }
+    } as ExistingBusinessInfoIF
+    store.stateModel.entityType = CorpTypeCd.ULC_CONTINUE_IN
+
+    const wrapper = wrapperFactory(ManualBusinessInfo)
```

Comment:
This test suite is about the ManualBusinessInfo component, so that's what I mount here. And below, I check that UploadAffidavit was rendered and contains the expected text (same as before).

Another way to do this is to create a separate test suite for UploadAffidavit, but I didn't bother.